Pasteboard Viewer/ContentView.swift

```diff
@@ -46,6 +46,7 @@ struct ContentView: View {
 				}
 					.labelsHidden()
 					.padding()
+				Button("Save to a file", action:{self.saveTofile(selectedType: self.selectedType)}).padding()
```

Comment:
This should be in the apps main menu, not in the window.

Reply:
From the original issue:

> We could add a "Save" menu item for that.

Reply:
Right, didn't understand that, I thought it was:  add save button the side menu, I'll change that